src/coreclr/ToolBox/superpmi/superpmi-shared/methodcontext.h

```diff
@@ -318,7 +318,7 @@ class MethodContext
         DWORD     sigTOK;
         DWORDLONG context;
     };
-    struct PInvokeMarshalingRequiredValue
+    struct MethodSigInfoPairValue
```

Comment:
This name is odd now. What is the pairing here?

Reply:
I'll rename this to be `MethodOrSigInfoValue` since it's not really pairing the two, but we need to encapsulate the two in a struct for superpmi.